Uplift/Controllers/Gym Detail/GymDetailViewController.swift

```diff
@@ -0,0 +1,184 @@
+
+        let edgeSwipe = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(back))
+        edgeSwipe.edges = .left
+        view.addGestureRecognizer(edgeSwipe)
+
+        setupViews()
+        setupConstraints()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        navigationController?.isNavigationBarHidden = true
+    }
+
+    // MARK: - Targets
+    @objc func back() {
+        navigationController?.popViewController(animated: true)
+    }
+}
+
+// MARK: - CollectionViewDataSource, CollectionViewDelegate, CollectionViewDelegateFlowLayout
+extension GymDetailViewController: UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return self.sections.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let itemType = sections[indexPath.item]
+
+        switch itemType {
+        case .hours:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: GymDetailHoursCell.reuseId, for: indexPath) as! GymDetailHoursCell
+            cell.configure(for: self, hours: gymDetail.hours, isDisclosed: gymDetail.hoursIsDisclosed)
+            return cell
+
+        case .tabbedController:
+            // swiftlint:disable:next force_cast
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: GymDetailTabbedControllerCell.reuseId, for: indexPath) as! GymDetailTabbedControllerCell
+            cell.configure(fitnessCenters: gymDetail.fitnessCenters)
+            cell.delegate = self
+            return cell
+        }
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let height: CGFloat
+
+        switch sections[indexPath.item] {
+        case .hours:
+            height = GymDetailHoursCell.getHeight(isDisclosed: gymDetail.hoursIsDisclosed, numLines: gymDetail.hours.getNumHoursLines())
+        case .tabbedController:
+            height = GymDetailTabbedControllerCell.getHeight(fitnessCenters: gymDetail.fitnessCenters[gymDetail.currentDisplayedFitnessCenter], viewWidt: self.view.frame.width)
+        }
+
+        return CGSize(width: collectionView.frame.width, height: height)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+        let headerView = collectionView.dequeueReusableSupplementaryView(
+            ofKind: UICollectionView.elementKindSectionHeader,
+            withReuseIdentifier: GymDetailHeaderView.reuseId,
+            // swiftlint:disable:next force_cast
```

Comment:
nit: remove

Reply:
This overrides the linter for force casting